code/modules/projectiles/guns/projectile/pistol.dm

```diff
@@ -1,5 +1,5 @@
 /obj/item/weapon/gun/projectile/automatic/pistol
-	name = "\improper FK-69 pistol"
+	name = "stechkin"

```

Comment:
the m1911 and deagle are explanatory by themselves, could pistol at least be tacked on to the end of this one?